### ISSUE TITLE
IE9/8 - cross-domain call from AJAX response not triggering

### ISSUE THREAD

**Issue by arunmenon1975** [jpillora/xdomain#158]
A link on my page (say, the homepage) makes an AJAX request and pulls in a form. The user enters data and submits the form to a PHP file. This PHP file returns data (echos) in the form of a `<script>`  chunk that has a soap object required by [jquery.soap](https://github.com/doedje/jquery.soap) with all the required params filled in. The URL to send the soap data(very similar to how the jquery `$.ajax`  URL is defined) is a cross-domain request. 

On IE10+, Chrome, Firefox and Safari and the stock Android browser Xdomain kicks in when it encounters the cross-domain URL in the AJAX response `<script>` and proceeds as desired. Everything works perfectly. 

On IE9/8 the `$.soap` URL isn't intercepted. The jquery.soap script gets everything correct right till the `beforeSend`. The next step would be to call the cross-domain URL, which simply just doesn't work in these 2 browser versions. I have logging turned on, and usually(in the browsers where it works) the xdomain logs would show the initiation of the cross-domain request and subsequent logs. 

I must be missing something obvious since IE9/8 are supported versions. There is a possibility that it could be an issue with the jquery.soap script, so i have [opened an issue](https://github.com/doedje/jquery.soap/issues/80) there as well.

**Comment by jpillora** [comment]
Hey @arunmenon1975 run through these https://github.com/jpillora/xdomain#faq--troubleshooting and report back (sorry, report back here, accidently posted on the soap repo)

**Comment by arunmenon1975** [comment]
I re-read the FAQs, went through a few issues listed, and made some changes. From early testing it now seems that the issue is fixed. 

A roundup of the changes:
- Made changes described at [issue #21 ](https://github.com/jpillora/xdomain/issues/21#issuecomment-36479577). This is mentioned in the FAQs as well. I still have not tested with having this change reversed.
- Commented out some lines (around 20, including comments) in the non-minified [jquery.soap](https://github.com/doedje/jquery.soap) script that referenced `xhr` for enabling progress info in the AJAX request (in the send function):

``` javascript
// second attempt to get some progres info (but still a no go)
// I still keep this in tho, we might see it working one day when browsers mature...
                xhr: function() {
                    var xhr = new window.XMLHttpRequest();
......
......
```
- added a catch-all `console` object script as IE sometimes returned errors that console wasn't defined. This may not be related to the fix, but i did get the error despite having it already defined in a short snippet. This error appeared off and on and was unpredictable.

Thanks for this fantastic script. It works out of the box and handles everything thrown at it. In my case, i had a few other things going in tandem requiring stringent observation under the hood.

Closing this issue for now.  I have also closed the [issue](https://github.com/doedje/jquery.soap/issues/80) opened at the jquery.soap repository.

_Please note: may re-open if issue persists while i test a little more intensively._
